lib/Mojo/Util.pm

```diff
@@ -14,9 +14,9 @@ use IO::Poll qw(POLLIN POLLPRI);
 use IO::Uncompress::Gunzip;
 use List::Util         qw(min);
 use MIME::Base64       qw(decode_base64 encode_base64);
+use Mojo::BaseUtil     qw(class_to_path monkey_patch);
 use Pod::Usage         qw(pod2usage);
 use Socket             qw(inet_pton AF_INET6 AF_INET);
-use Sub::Util          qw(set_subname);
```

Comment:
Do you think anyone depends on Mojo::Util::set_subname? They probably shouldn't but they could

Reply:
I would not consider use of any subroutines imported without a documented interface as our responsibility.

Reply:
No documentation, no feature.